templates/product/_product_collections.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+{% load staticfiles %}
+{% load price_range from price_ranges %}
+{% load product_first_image from product_images %}
+
+<h2>See also</h2>
+<hr>
+<div class="row">
+  {% for product in products %}
+      <div class="col-6 col-md-4 col-lg-2 product-list" itemscope itemtype="https://schema.org/Product">
```

Comment:
Not sure how useful that is since we're going to load it asynchronously. @mociepka?